l10n_it_edi_extension/data/invoice_it_template.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template
+        id="account_invoice_line_it_FatturaPA_inherit"
+        inherit_id="l10n_it_edi.account_invoice_line_it_FatturaPA"
+    >
+        <xpath expr="//DettaglioLinee/Natura" position="after">
+            <RiferimentoAmministrazione
+                t-if="it_values['admin_ref']"
+                t-out="it_values['admin_ref']"
+            />
+        </xpath>
+    </template>
+
+    <template
+        id="account_invoice_it_FatturaPA_export_inherit"
+        inherit_id="l10n_it_edi.account_invoice_it_FatturaPA_export"
+    >
+        <xpath expr="//CedentePrestatore/IscrizioneREA" position="before">
+            <StabileOrganizzazione t-if="company.l10n_edi_it_stable_organization">
+                <t t-set="partner" t-value="company.l10n_edi_it_stable_organization" />
+                <Indirizzo>
+                    <t
+                        t-if="partner.street or partner.street2"
+                        t-out="format_address(partner.street, partner.street2, 60)"
+                    />
+                </Indirizzo>
+                <CAP>
+                    <t t-out="partner.zip" />
+                </CAP>
+                <Comune t-out="format_alphanumeric(partner.city, 60)" />
+                <Provincia
+                    t-if="partner.state_id"
+                    t-out="format_alphanumeric(partner.state_id.code, 2)"
+                />
+                <Nazione t-out="partner.country_id.code" />
+            </StabileOrganizzazione>
+        </xpath>
+        <xpath expr="//CedentePrestatore/IscrizioneREA" position="after">
+            <RiferimentoAmministrazione
+                t-if="company.l10n_edi_it_admin_ref"
+                t-out="company.l10n_edi_it_admin_ref"
+            />
+        </xpath>
+        <xpath expr="//DatiGeneraliDocumento/ImportoTotaleDocumento" position="after">
+            <t t-foreach="causale" t-as="c">
+                <Causale t-out="c" />
```

Comment:
Anche questo è da documentare nel README

Reply:
Aggiunto in https://github.com/Borruso/l10n-italy/pull/28.